src/Libraries/DSOfficeNodes/CSVNodes.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace DSOffice
+{
+    /// <summary>
+    ///     Methods for Import/Export category.
+    /// </summary>
+    public static partial class Data
+    {
+        /// <summary>
+        ///     Write a list of lists into a file using a comma-separated values 
+        ///     format. Outer list represents rows, inner lists represent columns. 
+        /// </summary>
+        /// <param name="filePath">Path to write to</param>
+        /// <param name="data">List of lists to write into CSV</param>
+        /// <search>write,text,file</search>
+        public static void ExportCSV(string filePath, object[][] data)
```

Comment:
so I think this is a problem. You have removed the original assembly so you can no longer use type forwarding to redirect callers to the new assembly location.

This will likely break python and c# usage of these public methods and it may also break graph usage. What about continuing to build the DSOffice project as facade only even for net6?

Also please verify graphs containing these nodes open correctly after the reorganization.

Reply:
But I kept the same output name. 
DSOfficeNodes/DSOffice.csproj => will produce DSOffice.dll
I hoped that would allow typeforwarding

Reply:
Yes, the same fully qualified name